source/extensions/filters/http/aws_lambda/BUILD

```diff
@@ -0,0 +1,41 @@
+licenses(["notice"])  # Apache 2
+
+# L7 HTTP AWS Lambda filter
+# Public docs: docs/root/configuration/http_filters/aws_lambda_filter.rst
```

Comment:
This file doesn't seem to exist.

Reply:
I hadn't worked on documentation yet. Was leaving that to the end. But I've added it. Thanks.